doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+
+- global_matching.py
+- local_resampling.py
+
+.. note::
+    We highly recommend using the options described in :ref:`rerunning`. The speed bottleneck for the first diagnostic is the preprocessor. In the second diagnostic, step 1 is most time consuming, whereas steps 2 and 3 are likely to be repeated several times. Therefore, intermediate files are saved after step 1, and the diagnostic will automatically detect and use them if the ``-i`` flag is used.
+
+User settings
+-------------
+
+Datasets: Datasets have been split in two parts: the CMIP datasets and the target model datasets. An example use case for this recipe is to compare between CMIP5 and CMIP6, for example. The recipe can work with a target model that is not part of CMIP, provided that the data are CMOR compatible, and using the same data referece syntax as the CMIP data. Note that you can specify :ref:`multiple data paths<config-user-rootpath>` in the user configuration file.
+
+Preprocessors: We've tried to use built-in preprocessor functions as much as possible. The first diagnostic requires global mean temperature anomalies for each dataset, both CMIP and the target model, and some multimodel statistics. The second diagnostic requires data on a point in the Netherlands. However, the ``extract_point`` preprocessor can be changed to ``extract_shape`` or ``extract_region``, in conjunction with an area mean. And of course, the coordinates can be changed to analyze a different region.
+
+Diagnostics:
+
+* global_matching
+
+  * ``scenario_years``: a list of time horizons. Default: ``[2050, 2085]``
+  * ``scenario_percentiles``: a list of percentiles for the steering table. Default: ``[p10, p90]``
+
+* local_resampling
+
+  * ``control_period``: the control period shared between all scenarios. Default: ``[1981, 2010]``
+  * ``n_samples``: the final number of recombinations to be selected. Default: ``8``
+  * ``scenarios``: a scenario name and list of options. The default setting is a single scenario:
+
+    .. code-block:: yaml
+
+        scenarios:
+          ML_MOC:  # scenario name; can be chosen by the user
+            description: "Moderate / low changes in seasonal temperature & precipitation"
+            global_dT: 1.0
+            scenario_year: 2050
+            resampling_period: [2021, 2050]
+            dpr_winter: 4
+            pr_summer_control: [25, 55]
+            pr_summer_future: [45, 75]
+            tas_winter_control: [50, 80]
+            tas_winter_future: [20, 50]
+            tas_summer_control: [0, 100]
+            tas_summer_future: [0, 50]
+
+    These values are taken from table 1 in the Lenderink 2014's supplementary material. Multiple scenarios can be processed at once by appending more configurations below the default one. For new applications, ``global_dT``, ``resampling_period`` and ``dpr_winter`` are informed by the output of the first diagnostic. The percentile bounds are to be tuned until a satisfactory selection is achieved.
```

Comment:
Maybe for the last sentence: "The percentile bounds in the scenario settings (e.g. tas_winter_control and tas_winter_future) are to be tuned until a satisfactory scenario spread over the full CMIP ensemble is achieved."